api/v1/storagecluster_types.go

```diff
@@ -211,13 +211,10 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ManageCephToolbox defines how to reconcile Ceph toolbox & the placement configuration
 type ManageCephToolbox struct {
-	ReconcileStrategy string `json:"reconcileStrategy,omitempty"` // default behavior here is effectively `ignore`
-
-	// Tolerations if specified set toolbox ceph tools pod tolerations
-	// Defaults to empty
-	// +optional
-	Tolerations []corev1.Toleration `json:"cephTolerations,omitempty"`
+	ReconcileStrategy string               `json:"reconcileStrategy,omitempty"`
+	Placement         rookCephv1.Placement `json:"placement,omitempty"`
```

Comment:
We need to move this to `StorageClusterSpec.Placement` if we want to be consistent with other components.
None of the Manage* struct have a placement spec inside it.

Reply:
So we can just take the Placement info from the Storagecluster Spec.Placement.
We can take the tolerations out of that & apply that onto the toolbox. That would work right?
We would no longer need this field for Placement/Toleration inside the ManageCephToolBox.

Reply:
Done